### PR TITLE
Readcov

### PR DESCRIPTION
**Pull request by DLBPointon**

### CODE REVIEW

File: assets/local_testing/nxOscDF5033.yaml
Comment (weaglesBio) [inline assets/local_testing/nxOscDF5033.yaml:1]: Will need to apply this update to all test .yaml

File: conf/modules.config
Comment (weaglesBio) [inline conf/modules.config:1]: Will MINIMAP2_ALIGN_ILLUMINA need to be included in this?

File: subworkflows/local/yaml_input.nf
Comment (weaglesBio) [inline subworkflows/local/yaml_input.nf:1]: Is this necessary if same name? (read_ch)

File: conf/modules.config
Comment (weaglesBio) [reply conf/modules.config:1]: Also MINIMAP2_ALIGN_SE

**Review by weaglesBio — CHANGES_REQUESTED** [review]

**Comment by DLBPointon** [comment]
linting failing you need to run `nf-core modules patch` to fix

**Comment by yumisims** [comment]
Ok I will run and resubmit
thanks

From: Damon-Lee Pointon ***@***.***>
Reply to: sanger-tol/treeval ***@***.***>
Date: Wednesday, 10 January 2024 at 14:12
To: sanger-tol/treeval ***@***.***>
Cc: Ying Sims ***@***.***>, Assign ***@***.***>
Subject: Re: [sanger-tol/treeval] Readcov (PR #193) [EXT]


linting failing you need to run nf-core modules patch to fix

—
Reply to this email directly, view it on GitHub [github.com]<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_sanger-2Dtol_treeval_pull_193-23issuecomment-2D1884925168&d=DwMCaQ&c=D7ByGjS34AllFgecYw0iC6Zq7qlm8uclZFI0SqQnqBo&r=X9VPYUGy4ibZarh9Rup1XaZ62i2u-I0xOZXEy11dXDM&m=Zf3BV4rrlwX-8B-At8EgG2q2joumnQmeRgBi0TUyRg8zYi9zyczGHsZUSpxSW0jk&s=dfpIplo6BuMexXCzD9FlVM72RRi4F0QPsc5WL4CEnT4&e=>, or unsubscribe [github.com]<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_ADHNWY5E3MSF4HITI4RJSVTYN2OSXAVCNFSM6AAAAABBU2XD6CVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTQOBUHEZDKMJWHA&d=DwMCaQ&c=D7ByGjS34AllFgecYw0iC6Zq7qlm8uclZFI0SqQnqBo&r=X9VPYUGy4ibZarh9Rup1XaZ62i2u-I0xOZXEy11dXDM&m=Zf3BV4rrlwX-8B-At8EgG2q2joumnQmeRgBi0TUyRg8zYi9zyczGHsZUSpxSW0jk&s=KSrJCiSzG71nHYBtmplLO-gAeFmxfAMl1IO_mkpSOyA&e=>.
You are receiving this because you were assigned.Message ID: ***@***.***>



-- 
 The Wellcome Sanger Institute is operated by Genome Research 
 Limited, a charity registered in England with number 1021457 and a 
 company registered in England with number 2742969, whose registered 
 office is Wellcome Sanger Institute, Wellcome Genome Campus, 
 Hinxton, CB10 1SA

File: subworkflows/local/yaml_input.nf
Comment (yumisims) [reply subworkflows/local/yaml_input.nf:1]: better to be

File: conf/modules.config
Comment (yumisims) [reply conf/modules.config:1]: se and pe are not needed in the workflow any longer

File: conf/modules.config [hunk 1]
```diff
@@ -179,56 +178,51 @@ process {
...
     //
-    withName: ".*:.*:LONGREAD_COVERAGE:MINIMAP2_ALIGN" {
-            ext.args    = "--MD -t 8"
-            ext.prefix  = { "${meta.id}_alignment_${reference.getName().tokenize(".")[0]}" }
-    }
-
-    withName: ".*:.*:LONGREAD_COVERAGE:MINIMAP2_ALIGN_SPLIT" {
-            ext.args    = { "-t 20 --split-prefix ${meta.split_prefix}" }
-            ext.prefix  = { "${meta.id}_alignment_${reference.getName().tokenize(".")[0]}" }
+    withName: MINIMAP2_ALIGN {
```
[the remark this answers is not in the document]
Comment (yumisims) [reply conf/modules.config:183]: it has been changed

**Pull request merged**